### PR TITLE
[4.0] Fix com_tags view not found error

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue #16438.

### Summary of Changes
This PR fix view not found error reported at https://github.com/joomla/joomla-cms/issues/16438

### Testing Instructions
1. Follow the issue description, see the error
2. Apply patch, confirm that the error is fixed

### Additional comments

We actually should not have to set default_view in controller like this.  When default_view is not provided, it should be default to component name (which is tags in this case).  If https://github.com/joomla/joomla-cms/pull/15301 is reviewed, it should help solving this requirement

### CODE REVIEW

**Comment by joomdonation** [comment]
Drone error is not related to the change in this PR.

File: libraries/bootstrap.php [hunk 1]
```diff
@@ -95,9 +95,9 @@
 JLoader::registerAlias('JAdministrator', 'JApplicationAdministrator');
 JLoader::registerAlias('JSite', 'JApplicationSite');
 
-//JLoader::register('JNamespacePsr4Map', JPATH_LIBRARIES . '/namespacemap.php');
-//$extensionPsr4Loader = new JNamespacePsr4Map;
-//$extensionPsr4Loader->ensureMapFileExists();
+/* JLoader::register('JNamespacePsr4Map', JPATH_LIBRARIES . '/namespacemap.php');
+   $extensionPsr4Loader = new JNamespacePsr4Map;
+   $extensionPsr4Loader->ensureMapFileExists(); */
```
Comment (Bakual) [inline libraries/bootstrap.php]: Unrelated change

File: libraries/namespacemap.php [hunk 2]
```diff
@@ -23,6 +23,11 @@ class JNamespacePsr4Map
 	 */
 	protected $file = '';
 
+	/**
+	 * Class constructor
+	 *
+	 * @since  __DEPLOY_VERSION__
```
Comment (Bakual) [inline libraries/namespacemap.php]: Unrelated change

File: libraries/bootstrap.php (hunk 1 above)
Comment (joomdonation) [inline libraries/bootstrap.php]: Yes. It is CS error and need to be fixed.

File: libraries/namespacemap.php (hunk 2 above)
Comment (joomdonation) [inline libraries/namespacemap.php]: Same reason as above

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 151103fec0631a9e66350de797591b1cdf569d6c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16443">issues.joomla.org/tracker/joomla-cms/16443</a>.</sub>

**Comment by coolcat-creations** [comment]
I have tested this item :white_check_mark: successfully on 151103fec0631a9e66350de797591b1cdf569d6c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16443">issues.joomla.org/tracker/joomla-cms/16443</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by wilsonge** [comment]
Can we remove the code style stuff. That will be fixed by #16490

**Comment by joomdonation** [comment]
Sorry.  I opened #16491 to replace this one, somehow it has merge conflict on my local repo and I could not fix it
